### ISSUE TITLE
Milestone 4: Peer review feedback

### ISSUE THREAD

**Issue by sean-m-mckay** [UBC-MDS/wine-origin-prediction#55]
**From John**
- It is better to save the raw data explicitly into /data/raw/, after from ucimlrepo import fetch_ucirepo and data = fetch_ucirepo(id=109), so that we are able to read the raw data without running the codes 
- The report would look nicer if you could hide the codes at the beginning 
 - It would be a big plus if an example is provided at the end of the report, to illustrate some predictions made by the Logistic Regression, and some discussions to reinforce how the predictions benefit the industry

**From tangyl92**
- It would be useful to include more detailed results from the model, such as confusion matrices or ROC curves, to better understand its performance.
- Discussing the significance of the observed class imbalance and its potential impact on model performance would add depth to your analysis.
- It would be immensely beneficial if you could provide the coefficient values and their corresponding interpretations from your model.

**From Doris**
- Dependencies section on README clearly listed dependencies required for running the project within the local environment using environment.yml. As there is another option, the docker container, created to run this project, it would be better to include the dependencies on dockerfile in this section.

   - Main report clearly includes all references related to the project and figures are properly generated on the report. It would be nice if the report can also include captions for each figure, as well as the figure number for reference within the analysis, which would be easier for users to follow the analysis. Also, hiding the code cells on the main report may reduce distractions from the main analysis that the authors want to deliver.

    - Looks like the project includes hyperparameter tuning as it includes a line chart of accuracy vs hyperparameter C for training set and validation set. May be more comments on what value of hyperparameter C is chosen for the best model would help the audience better understand the model chosen for this project.

    - I wasn’t able to reproduce the report in the container. I successfully opened up the container using docker compose up of setup option 1. However, when I tried to run multiple scripts listed in the data analysis section and tests using pytest, I encountered an error of a module not found. I wonder if the dockerfile copies dependencies properly from the environment.yml file or my user mistakes. I may need some help from the project team to guide me how to run these scripts properly.

**Comment by hbandukw** [comment]
It is better to save the raw data explicitly into /data/raw/, after from ucimlrepo import fetch_ucirepo and data = fetch_ucirepo(id=109), so that we are able to read the raw data without running the codes - DONE!  https://github.com/UBC-MDS/wine-origin-prediction/pull/54
